scipy/sparse/__init__.py

```diff
@@ -10,14 +10,15 @@
 
    sparse.csgraph
    sparse.linalg
+   sparse.migration_to_sparray
 
 SciPy 2-D sparse array package for numeric data.
 
 .. note::
 
    This package is switching to an array interface, compatible with
    NumPy arrays, from the older matrix interface.  We recommend that
-   you use the array objects (`bsr_array`, `coo_array`, etc.) for
+   you use the array objects (``bsr_array``, ``coo_array``, etc.) for
```

Comment:
I think we still want single-backticks here, as those will produce links to the class definitions.